rev_news/drafts/edition-15.md

```diff
@@ -248,6 +248,7 @@ which is very useful when rebasing a large patch series.  I also have to mention
 
 __Various__
 
+* BitKeeper, the inspiration for Git and Mercurial, [was released as Open Source](https://users.bitkeeper.org/t/bk-7-2ce-released-2016-05-09/93) (see also the discussion on [Hacker News](https://news.ycombinator.com/item?id=11667494) as well as [Linux Weekly News](https://lwn.net/Articles/686896/)).
```

Comment:
I am  not sure "inspiration" is a good word here. I do not know about Hg, but I remember that Linus said that he deliberately designed Git to be different from BitKeeper to avoid any confusion.

The action of BitMover the company did serve as a trigger to make people aware of the lack of good open source DVCS, which led Git and other systems getting widely and quickly accepted, but that is ironically a negative influence.